server/services/interventionsService.ts

```diff
@@ -210,6 +210,20 @@ export default class InterventionsService {
     })) as DraftReferral
   }
 
+  async amendComplexityLevelForServiceCategory(
+    token: string,
+    referralId: string,
+    complexityLevel: Partial<ReferralComplexityLevel>
+  ): Promise<AmendReferralDetailsUpdate> {
```

Comment:
we are going to return 204 no content for the update endpoints, so this return type should be Promise<null>